src/test/kotlin/org/rust/ide/refactoring/RsExtractEnumVariantTest.kt

```diff
@@ -476,9 +479,20 @@ class ExtractEnumVariantIntentionTest : RsIntentionTestBase(ExtractEnumVariantIn
                 V2
             }
         }
-        
+
         fn main() {
             let _ = E::V1(V1 { x: 0, y: 1 });
         }
     """)
+
+    private fun doAvailableTest(@Language("Rust") before: String, @Language("Rust") after: String) {
+        InlineFile(before.trimIndent()).withCaret()
+        myFixture.testAction(RsExtractEnumVariantAction())
+        myFixture.checkResult(replaceCaretMarker(after.trimIndent()))
+    }
```

Comment:
Can be simplified to

```suggestion
    private fun doAvailableTest(@Language("Rust") before: String, @Language("Rust") after: String) {
        checkEditorAction(before, after, "Rust.RsExtractEnumVariant")
    }
```